src/Apps/Order/Utils/offerUtils.ts

```diff
@@ -0,0 +1,127 @@
+import { DEFUALT_OFFER_NOTE_PREFIX } from "Apps/Order/Routes/Offer"
+import { PriceOptions_artwork$data } from "__generated__/PriceOptions_artwork.graphql"
+
+export interface OfferItem {
+  price: string
+  displayPriceRange?: boolean
+}
+
+type ListPriceType = PriceOptions_artwork$data["listPrice"] | undefined
+type ListPriceOptionsType = (
+  | {
+      key: string
+      value: number
+      description: string
+    }
+  | undefined
+)[]
+
+export const getOfferItemFromOrder = (orderLineItemsNode): OfferItem | null => {
+  const offerItem = orderLineItemsNode.edges[0].node.artworkOrEditionSet
+
+  if (["Artwork", "EditionSet"].includes(offerItem?.__typename)) {
+    return offerItem
+  }
+  return null
+}
+
+export const lastOfferNote = (note: string) => {
+  if (note.startsWith(DEFUALT_OFFER_NOTE_PREFIX)) {
+    return ""
+  } else {
+    return note
+  }
+}
+
+export const getInitialOfferState = (
+  listPriceOptions: ListPriceOptionsType,
+  lastOffer?: number
+) => {
+  let lastOfferValue: number | undefined = undefined
+  let selectedPriceOption = listPriceOptions[0]?.key
+  let selectedPriceValue = listPriceOptions[0]?.value
+
+  if (lastOffer) {
+    const matchingPriceOption = listPriceOptions.find(
+      option => option?.value === lastOffer
+    )
+
+    if (matchingPriceOption) {
+      selectedPriceOption = matchingPriceOption.key
+      selectedPriceValue = matchingPriceOption.value
+    } else {
+      lastOfferValue = lastOffer
+      selectedPriceOption = "price-option-custom"
+      selectedPriceValue = undefined
+    }
+  }
+
+  return {
+    lastOffer: lastOfferValue,
+    selectedPriceOption,
+    selectedPriceValue,
+  }
+}
+
+export const getOfferPriceOptions = (
+  listPrice: ListPriceType,
+  isPriceRange?: boolean | null
+) => {
+  return isPriceRange
+    ? getRangeOptions(listPrice)
+    : getPercentageOptions(listPrice)
+}
+
+const getRangeOptions = (listPrice: ListPriceType) => {
+  const minPriceRange = listPrice?.minPrice?.major
+  const maxPriceRange = listPrice?.maxPrice?.major
+  const midPriceRange = Math.round(
+    (Number(minPriceRange) + Number(maxPriceRange)) / 2
+  )
+  const getRangeDetails = [
+    {
+      value: maxPriceRange,
+      description: "Top-end of range (high chance of acceptance)",
+    },
+    {
+      value: midPriceRange,
+      description: "Midpoint (good chance of acceptance)",
+    },
+    {
+      value: minPriceRange,
+      description: "Low-end of range (lower chance of acceptance)",
+    },
+  ]
+
+  return getRangeDetails.map((rangePrice, idx) => ({
+    key: `price-option-${idx}`,
+    value: rangePrice.value!,
+    description: rangePrice.description,
+  }))
```

Comment:
Since there are only 3 options (and we don't foresee adding more), this method would be easier to read if it just returned lines 82-93 with the `key` field added to each and the `value` field populated inline.

I also think that the `key` for each option can be more meaningful (ie. price-option-min instead of price-option-2).